sdk/keyvault/azure-keyvault-secrets/samples/list_operations.py

```diff
@@ -96,7 +97,7 @@ def run_sample():
 
     except HttpResponseError as e:
         if "(NotSupported)" in e.message:
```

Comment:
Not related to the PR, but we should *really* avoid having samples that parses exception messages! Please open an issue to address!

Reply:
#8405